mscore/CMakeLists.txt

```diff
@@ -680,6 +680,12 @@ if (APPLE)
       REGEX ".*_debug\\.dylib" EXCLUDE)
 endif (APPLE)
 
+# Add the appropriate splash screen for the build
+if (MSCORE_UNSTABLE)
+    cp -f /assets/splash-dev.png  mscore/data/splash.png # Dev splash on development builds
```

Comment:
CMake is not a shell program, so I don't think `cp` will work outside a custom command, and may not work on all platforms anyway. Try [this](https://stackoverflow.com/a/34800667) or [this](https://github.com/musescore/MuseScore/blob/e1214dfa6186193b16204e099538ae08cd349d1a/mscore/CMakeLists.txt#L625) instead.

Notice that when CMake is run the whole source directory is basically copied into `build.release` (or `build.debug`), which is known as `${PROJECT_BINARY_DIR}`. The splash probably needs to end up in `${PROJECT_BINARY_DIR}/mscore/data` rather than `${PROJECT_SOURCE_DIR}/mscore/data`. CMake might do this for you whereas `cp` definitely wont.

Reply:
Oh, thanks! I'll fix that now.

Reply:
OK, the command is definitely:

```cmake
configure_file(/assets/splash-dev.png  mscore/data/splash.png COPYONLY)
```

CMake knows to put it in `${PROJECT_BINARY_DIR}`.

This won't work...

```cmake
install(FILES /assets/splash-dev.png  RENAME splash.png DESTINATION /mscore/data)
```

...because `install` is run as part of the install step (i.e. after compilation is complete), which is too late for things that need to go *inside* the executable, but which is the [right time to copy the file icons](https://github.com/musescore/MuseScore/blob/e1214dfa6186193b16204e099538ae08cd349d1a/mscore/CMakeLists.txt#L625) since they do not go inside the executable.

Reply:
Oops! those paths should have been relative, not absolute... Sorry!

Reply:
Is it relative for both or just the first?